source/PlayerInfo.cpp

```diff
@@ -709,9 +738,24 @@ const CoreStartData &PlayerInfo::StartData() const noexcept
 
 
 
+void PlayerInfo::SetSystemEntry(const SystemEntry entryType)
```

Comment:
The header file lacks this const.
```suggestion
void PlayerInfo::SetSystemEntry(SystemEntry entryType)
```